.github/workflows/backend-integration-test.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           name: integrate test reports
           path: |
             build/reports
-            integration-test/build/integration-test.log
+            integration-test/build/integration-test-integration-test.log
```

Comment:
Why do we need two "integration-test"?

Reply:
The log name is `param.systemProperty("gravitino.log.path", project.buildDir.path + "/${project.name}-integration-test.log")` for all modules that have integration test and the project name is `integrate-test`

Reply:
I see.